modules/virt-define-http-readiness-probe.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+[id="virt-define-http-readiness-probe_{context}"]
+
+= Defining an HTTP readiness probe
+
+Define an HTTP readiness probe by setting the `spec.readinessProbe.httpGet` field of the virtual machine instance (VMI) configuration.
+
+
+.Procedure
+. Include details of the readiness probe in the VMI configuration file.
++
+
+.Sample readiness probe with an HTTP GET test
+[source,yaml]
+----
+spec:
+  readinessProbe:
+    httpGet: <1>
+      port: 1500 <2>
+    initialDelaySeconds: 120 <3>
+    periodSeconds: 20 <4>
+    timeoutSeconds: 10 <5>
+    failureThreshold: 3 <6>
+    successThreshold: 3 <7>
+----
+<1> The HTTP GET request to perform to connect to the VMI.
+<2> The port of the VMI that the probe queries. In the above example, the probe queries port 1500. The VMI installs and runs a minimal HTTP server on port 1500 via cloud-init.
```

Comment:
`The VMI installs and runs a minimal HTTP server on port 1500 via cloud-init.`
This is not true in the example as the cloud-init part is missing. It's only shown below.

Reply:
Will remove that line